warrior/Framework/Utils/email_utils.py

```diff
@@ -163,6 +163,7 @@ def send_email(smtp_host, sender, receivers, subject, body, files):
     message = MIMEMultipart()
     message['From'] = sender
     message['To'] = receivers
+    receivers_list = [receiver.strip() for receiver in receivers.split(',')]
```

Comment:
The delimiter for e-mails in outlook is ";" semi-colon gmail accepts both comma and semicolon. 
This comment has no impact on overall review.